Content.Server/Chemistry/Metabolism/OverdoseMetabolism.cs

```diff
@@ -0,0 +1,63 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Content.Shared.Damage;
+using Content.Shared.Chemistry.Metabolizable;
+using Content.Shared.Chemistry.Reagent;
+using Content.Shared.Damage.Components;
+using Content.Server.Chemistry.Components;
+
+namespace Content.Server.Chemistry.Metabolism
+{
+    /// <summary>
+    /// Deals a certain amount of damage once a certain amount of reagent is ingested at once.
+    /// </summary>
+    [DataDefinition]
+    public class OverdoseMetabolism : IMetabolizable
+    {
+        /// <summary>
+        /// How much of the reagent should be metabolized each sec.
+        /// </summary> 
+        [DataField("rate")]
+        public ReagentUnit MetabolismRate { get; set; } = ReagentUnit.New(1);
```

Comment:
if you don't want to do the generalized overdose thing (which you really should, or just don't do overdoses) then this should at least be 0, doesn't it being 1 mean that the reagent metabolizes twice as fast as it should even if you don't overdose